splice_machine/src/main/java/com/splicemachine/derby/ddl/DDLWatchRefresher.java

```diff
@@ -72,6 +71,19 @@ public DDLWatchRefresher(DDLWatchChecker watchChecker,
         ddlDemarcationPoint = new AtomicReference<>();
     }
 
+    public void getDemarcationPoint() {
```

Comment:
This should be called `initDemarcationPoint` I think

Reply:
Right, it is renamed